build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '2.3.3.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+}
+
+group = 'com.cag.cagbackendapi'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+	implementation "org.apache.tomcat.embed:tomcat-embed-core:9.0.35"
+	implementation "org.springframework.boot:spring-boot-starter-data-jpa"
+	implementation "com.nimbusds:nimbus-jose-jwt:8.2.1"
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation "org.springframework.retry:spring-retry:1.2.4.RELEASE"
+	implementation "org.springframework.boot:spring-boot-starter-aop:1.5.22.RELEASE"
+	implementation "org.springframework.boot:spring-boot-starter-actuator"
+	implementation "org.springframework.boot:spring-boot-starter-data-rest"
+	implementation 'junit:junit:4.12'
+	implementation "org.springframework:spring-web:5.2.3.RELEASE"
+	implementation "org.dom4j:dom4j:2.1.3"
+	implementation "com.fasterxml.jackson.module:jackson-module-kotlin:2.10.0"
+	implementation "org.jetbrains.kotlin:kotlin-reflect:1.3.50"
+	implementation "com.google.code.gson:gson:2.8.5"
+	//implementation "org.flywaydb:flyway-core"
```

Comment:
No need to leave commented out code in

Reply:
Yes, we do need that for the future. I anticipate our database schema changing many times as our project grows and requirements change. flyway is a database migration tool that will help us alter our tables in a structured format that makes it easy to rollback changes if we alter something wrong and it breaks. If I uncomment this then the project won't be able to run because it will look for all the flyway configuration files I haven't added yet. We'll have this working soon!